frames/src/main/java/com/checkout/frames/di/component/FramesDIComponent.kt

```diff
@@ -45,6 +47,12 @@ internal abstract class FramesDIComponent {
         @BindsInstance
         fun cardTokenizationUseCase(cardTokenizationUseCase: UseCase<InternalCardTokenRequest, Unit>): Builder
 
+        @BindsInstance
+        fun closePaymentFlowUseCase(
+            @Named(CLOSE_PAYMENT_FLOW_DI)
+            closePaymentFlowUseCase: UseCase<Unit, Unit>
+        ): Builder
+
```

Comment:
Can we use separate  Named annotations for both usecase we are using for closePayment Usecase and cardTokenizationUseCase?

Reply:
`@Named` annotation, in this case, is needed because `UseCase<Unit, Unit>` is not self-representative, you can’t even make an assumption from generic types what it does.